### ISSUE TITLE
Figure missing ...

### ISSUE THREAD

**Issue by ssavvides** [oasislabs/docs#57]
.. @ https://github.com/oasislabs/docs#new-devnet-new-tools

**Comment by nhynes** [comment]
That's okay. The readme in this repo isn't meant for public consumption. It's just what [vuepress displays as the home page](https://vuepress.vuejs.org/default-theme-config/#homepage) on our docs site.

**Comment by omitar** [comment]
I do not think you can control what is used by public. If you do not want README.md to be read, then make it a simple link to rendered documentation and move the front page for vuepress to some subdirectory of the repository.

**Comment by nhynes** [comment]
> move the front page for vuepress to some subdirectory of the repository.

Do you think that I didn’t want to do that? Obviously I would have if it were possible to do without forking the vuepress repo.

Please try to be more humble with your suggestions. You’re working with other capable people and, in many cases, you can trust that they care just as much—if not more—about user experience. What you’re doing right now is low-effort and disrespectful of the effort that’s already gone into what exists.
